Simplified/NYPLAccountSignInViewController.m

```diff
@@ -590,6 +591,17 @@ + (void)authorizeUsingExistingBarcodeAndPinWithCompletionHandler:(void (^)())han
 
 #pragma mark -
 
+- (void)scanLibraryCard
+{
+  [NYPLBarcode presentScannerWithCompletion:^(NSString * _Nullable resultString) {
```

Comment:
is running presentScannerWithCompletion guaranteed to not throw an error?

Reply:
resultString is `_Nullable` which I use as check for if the method failed or not. Methods that "throw" errors are enforced at compile time. Longer discussion is below.